### DISCUSSION TITLE
Possible to read consumption and PV?

### DISCUSSION THREAD

**Discussion by Friedi1970** [klaasnicolaas/home-assistant-glow#176] [General]
I am really interested in this little thing but I wonder whether my meter is compatible.
My power meter shows both, the consumption of me (the power I get) as well as the power I send (produced with PV). Does anybody know this?

**Comment by xsaMNJfkltrwq5r3** [comment]
Does your meter have two independant pulse LEDS, one LED for importing and the other for exporting?

**Reply by realfire23** [reply]
hi,

I have the same question but I only have 1 led

**Reply by klaasnicolaas** [reply]
It will not really work with 1 LED, your meter will probably calculate what goes in and out and determines the behavior of your pulse LED based on that.

**Reply by snoopydooper** [reply]
Mine has only one LED but I believe the pulse is slightly longer for export/generation. Is this workable?

**Reply by klaasnicolaas** [reply]
This depends very much on your situation, the moment you return more than you consume then it will not really work well anymore.
